templates/components/login-no-pass-form.jsx

```diff
@@ -18,8 +18,8 @@ var LoginNoPassword = React.createClass({
   render: function() {
     return (
       <div className="loginNoPass innerForm centerDiv">
-        <Form origin="Migration" ref="userform" fields={fields} validators={fieldsValidators} />
-        <button onClick={this.processFormData} className="btn btn-awsm">Log in</button>
+        <Form origin="Migration" ref="userform" fields={fields} validators={fieldsValidators} defaultUsername={this.props.username} />
```

Comment:
I think I lost that when we closed the vidyo, but I remembered that we need to change it after you've sent the first copy because of some reason... >.<